### PR TITLE
Fix python django app

### PR DESCRIPTION
**Pull request by quoctruong**
Python Django App is not configured for debugging so we have to add support for it.

### CODE REVIEW

File: python/django/python-guestbook/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "cloudcode.profile-cluster-map": {
+        "[default]": "another-context"
```
Comment (simonz130) [inline python/django/python-guestbook/.vscode/settings.json]: can we use more meaningful name?

File: python/django/python-hello-world/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "cloudcode.profile-cluster-map": {
+        "[default]": "another-context"
```
Comment (simonz130) [inline python/django/python-hello-world/.vscode/settings.json]: Please remove the settings file

File: python/django/python-guestbook/.vscode/settings.json (hunk 1 above)
Comment (simonz130) [reply python/django/python-guestbook/.vscode/settings.json]: or actually, remove the settings file changes altogether

**Comment by quoctruong** [comment]
@simonz130 PTAL

**Review by simonz130 — APPROVED** [review]

**Pull request merged**